setup.py

```diff
@@ -53,6 +53,7 @@
         'pyasn1>=0.4.0,<0.5.0;python_version<"3.0"',
         'pyasn1-modules>=0.2.0,<0.3.0;python_version<"3.0"',
         'enum34;python_version<"3.4"',
+        'urllib3>=1.21.1,<1.25',
```

Comment:
can you limit only for Python 3.4 or less?

Reply:
Test the package locally before sending the change.

```
cd ~/snowflake/Python
./build_connector.sh -c
python3.4 -m venv /tmp/p
source /tmp/p/bin/activate
pip install dist/snowflake_connector*
```
No error or warning should show up.

Reply:
would not this work?
```
'urllib3>=1.21.1,<1.25;python_version<"3.5"',
```